### PR TITLE
addressing dmueller's comments

### PR DESCRIPTION
**Pull request by dshimo**
continuation of https://github.com/OpenLiberty/blogs/pull/955

### CODE REVIEW

File: posts/2020-12-15-openliberty-realworld-example-app.adoc [hunk 1]
```diff
@@ -14,15 +14,15 @@ David Shi <https://github.com/dshimo>
...
+Let link:https://github.com/gothinkster/realworld[RealWorld] serve as your fitting room for playing around and building "real world" applications. Created as an open source project, RealWorld enables users to choose any combination of frontend and backend to see how they work together in a sample app called link:https://demo.realworld.io/["Conduit"]. This provides developers a great sandbox environment for exploring new or familiar technologies. What if you're a frontend developer with no interest in backend, or vice versa? Since the frontend or backend can run independently of each other, you can for example, dedicate your efforts to finding the frontend that suits you without knowing any details of how the backend performs its tasks. RealWorld manages this by enforcing link:https://github.com/gothinkster/realworld/tree/master/spec#project-overview[API specs] for Conduit's endpoints.
```
Comment (dmuelle) [inline posts/2020-12-15-openliberty-realworld-example-app.adoc]: edit suggestions- avoid using "This" as the subject of a sentence. Can fix by answering "This what?". For example

This provides developers a great sandbox environment for exploring new or familiar technologies.
--->
This app provides developers a great sandbox environment for exploring new or familiar technologies.

RealWorld manages this by enforcing...
-->
RealWorld manages this  separation by enforcing

File: posts/2020-12-15-openliberty-realworld-example-app.adoc [hunk 2]
```diff
@@ -14,15 +14,15 @@ David Shi <https://github.com/dshimo>
...
 
-For today's blog post, I'm taking advantage of several link:https://www.openliberty.io/guides/[Open Liberty guides] to build the link:https://github.com/OpenLiberty/openliberty-realworld-example-app[backend server] for RealWorld by using Open Liberty and MicroProfile. 
+For today's blog post, I'll be exploring some MicroProfile technologies and link:https://www.openliberty.io/guides/[Open Liberty guides] for powering Conduit's backend on Open Liberty. Specifically, we'll be propping up a link:https://en.wikipedia.org/wiki/Jakarta_RESTful_Web_Services[JAX-RS] application for our REST endpoints, adding some security to them, and then persisting our data with link:https://en.wikipedia.org/wiki/Jakarta_Persistence[JPA].
```
Comment (dmuelle) [inline posts/2020-12-15-openliberty-realworld-example-app.adoc]: I'll be exploring
--->
I explore

Conduit's backend
--->
the Conduit backend

we'll be propping up... adding some security...then persisting
--->
we'll prop up... add some security... then persist

File: posts/2020-12-15-openliberty-realworld-example-app.adoc [hunk 3]
```diff
@@ -14,15 +14,15 @@ David Shi <https://github.com/dshimo>
...
 
 == Starting a RESTful App
 
-Let's begin by deploying a JAX-RS application with some endpoints so we can start playing with it. By following link:https://openliberty.io/guides/rest-intro.html[guide to create a RESTful web service], we can prop up an endpoint to return simple messages or JSON-formatted classes. As a simplified example, for our blogging website we can start with an endpoint to return an instantiated user:
+Let's begin by deploying a JAX-RS application with some initial endpoints so we can start playing with it. By following link:https://openliberty.io/guides/rest-intro.html[guide to create a RESTful web service], we can prop up an endpoint to return simple messages or JSON-formatted classes. As a simplified example, for our blogging website we can start with an endpoint to return an instantiated user:
```
Comment (dmuelle) [inline posts/2020-12-15-openliberty-realworld-example-app.adoc]: By following link:https://openliberty.io/guides/rest-intro.html[guide to create a RESTful web service]
--->
By following the link:https://openliberty.io/guides/rest-intro.html[guide to create a RESTful web service]

for our blogging website
what is the blogging website? It isn't mentioned elsewhere or explained- is it related to the conduit app?

File: posts/2020-12-15-openliberty-realworld-example-app.adoc [hunk 4]
```diff
@@ -131,6 +131,8 @@ public class UserDao {
 
 == Try it Out!
 
+You can pull the code in and play with it on your own machine. Our full implementation can be found on our link:https://github.com/OpenLiberty/openliberty-realworld-example-app[Open Liberty GitHub] where the README should cover how to build it locally. To view Conduit locally, I personally chose link:https://github.com/gothinkster/angular-realworld-example-app[Angular] as my frontend, and followed their instructions on starting it. Of course, you can choose whichever link:https://codebase.show/projects/realworld[frontend] you'd like!
```
Comment (dmuelle) [inline posts/2020-12-15-openliberty-realworld-example-app.adoc]: ...where the README should cover how to build it locally.
-->
...where the README explains how to build it locally.

...you'd like
-->
...you like

**Review by dmuelle — CHANGES_REQUESTED** [review]
Looks good! Just a few minor edit suggestions and a question to clarify mention of "blogging website"

File: posts/2020-12-15-openliberty-realworld-example-app.adoc (hunk 3 above)
Comment (dshimo) [reply posts/2020-12-15-openliberty-realworld-example-app.adoc]: yes Conduit is the blogging website. i've replaced that term with Conduit

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**